### ISSUE TITLE
Configure center align with odd numbered content

### ISSUE THREAD

**Issue by bartlett-sre** [dhruvasagar/vim-table-mode#223]
Is it possible to configure the center align behaviour so that it aligns one character left when the cell content is an odd number? Thanks, I'm loving this plugin!

**Default Behaviour**
```md
| Encoding |
|:--------:|
|   utf-8  |
```

**Desired Behaviour**
```
| Encoding |
|:--------:|
|  utf-8   |
```

**Comment by dhruvasagar** [comment]
@bartlett-sre If you are willing to submit a PR, I can point you in the right direction, it should be pretty easy

**Comment by bartlett-sre** [comment]
Yeah that would be awesome thanks! I had a quick look, but vimscript is pretty new to me.

**Comment by dhruvasagar** [comment]
@bartlett-sre https://github.com/dhruvasagar/vim-table-mode/blob/0dd8aff6d1690298617e652b0187059deae291c5/autoload/tablemode/align.vim#L9-L20, this function handles adding the padding, the third case is the one that handles center aligning.
